after/ftplugin/php.vim

```diff
@@ -20,6 +20,14 @@ if !exists('g:hack#hh_client')
   let g:hack#hh_client = 'hh_client'
 endif
 
+if !exists('g:hack#hh_format') && executable('hh_format')
+  let g:hack#hh_format = 'hh_format'
+endif
+
+if !exists('g:hack#root')
+  let g:hack#root = '~/www'
+endif
```

Comment:
Is the `--root` argument required?  If it defaults to cwd like hh_client, you should probably leave this unset, and use it only if it's set in the function.